File: Back/src/ProEventos.API/Models/Evento.cs
namespace ProEventos.API.Models
{
    public class Evento
    {
        public int EventoId { get; set; }
        public string Local { get; set; }
        public string DataEevento { get; set; }
        public string Tema { get; set; }
        public int QtdPessoas { get; set; }
        public string Lote { get; set; }
        public string ImagemURL { get; set; }

    }
}

File: Front/ProEventos-App/src/app/models/Evento.ts
import { Lote } from './Lote';
import { Palestrante } from './Palestrante';
import { RedesSociais } from './RedesSociais';

export interface Evento {
  id: number;
  local: string;
  dataEvento?: Date;
  tema: string;
  qtdPessoas: number;
  imagemURL: string;
  telefone: number;
  email: string;
  lotes: Lote[];
  redesSociais: RedesSociais[];
  palestrantesEventos: Palestrante[];
}
